src/components/MoviesSearchBar/MoviesSearchBar.js

```diff
@@ -36,8 +36,8 @@ const MoviesSearchBar = ({ searchQuery }) => {
 
   return (
     <div className="movies-search-bar">
-      <h1 className="movies-search-bar__header">Find your movie</h1>
-      <form className="movies-search-bar__form">
+      <h1 className="search-bar-header">Find your movie</h1>
+      <form className="search-bar">
```

Comment:
A form doesn't resemble the search bar.

```suggestion
      <form className="search-form">
```